vehicle_launch/launch/vehicle_interface.launch.xml

```diff
@@ -16,7 +16,7 @@
 
   <!-- real mode -->
   <group unless="$(var simulation)">
-    <include file="$(var vehicle_model_pkg)/launch/vehicle_interface.launch.xml">
+    <include file="$(var vehicle_model_pkg)/launch/vehicle_interface.launch.xml" pass_all_args="true">
```

Comment:
I guess pass_all_args is not supported in ros2 launch.

Reply:
fixed!

Reply:
@TakaHoribe It does not seem to be fixed.